src/test/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportControllerTest.java

```diff
@@ -69,7 +68,7 @@
 import static uk.gov.hmcts.cmc.domain.models.ClaimDocumentType.SETTLEMENT_AGREEMENT;
 import static uk.gov.hmcts.cmc.domain.models.response.YesNoOption.YES;
 
-@RunWith(MockitoJUnitRunner.class)
+@ExtendWith(MockitoExtension.class)
 public class SupportControllerTest {
```

Comment:
Documentation says - _Test classes, test methods, and lifecycle methods are not required to be public, but they must not be private._

I have removed public from class and methods but its not required i guess..

Reply:
It's generally best practice to restrict visibility as much as possible. It also has the benefit of making it very quickly obvious that the test class is written for JUnit 5 instead of 4. "Not required" is correct though!

Reply:
agree with what you say..